static/lib/composer/drafts.js

```diff
@@ -39,6 +38,8 @@ define('composer/drafts', function() {
 				});
 			}
 		});
+
+		drafts.migrateGuest();
```

Comment:
When you log in, sessonStorage drafts are saved to localStorage for the logged-in user